public/app/features/dashboard-scene/utils/dashboardSceneGraph.test.ts

```diff
@@ -1,68 +1,130 @@
-import { DashboardDataDTO } from 'app/types';
+import {
+  SceneGridItem,
+  SceneGridLayout,
+  SceneQueryRunner,
+  SceneRefreshPicker,
+  SceneTimePicker,
+  SceneTimeRange,
+  VizPanel,
+} from '@grafana/scenes';
 
-import dashboard_to_load from '../serialization/testfiles/dashboard_to_load1.json';
-import { transformSaveModelToScene } from '../serialization/transformSaveModelToScene';
+import { DashboardControls } from '../scene/DashboardControls';
+import { DashboardLinksControls } from '../scene/DashboardLinksControls';
+import { DashboardScene, DashboardSceneState } from '../scene/DashboardScene';
 
 import { dashboardSceneGraph } from './dashboardSceneGraph';
 
 describe('dashboardSceneGraph', () => {
   describe('getTimePicker', () => {
     it('should return null if no time picker', () => {
-      const dashboard: DashboardDataDTO = {
-        ...(dashboard_to_load as unknown as DashboardDataDTO),
-        timepicker: {
-          hidden: true,
-          refresh_intervals: [],
-          time_options: [],
-        },
-      };
-
-      const scene = transformSaveModelToScene({
-        dashboard: dashboard as unknown as DashboardDataDTO,
-        meta: {},
+      const scene = buildTestScene({
```

Comment:
Creating the scene for the tests, since we're adding too many dependencies in between if we rely on the serializer. Since the tests are for the graph util only, I decided to create the scene without passing through the serializer. The scenario is easier to set up.

Reply:
makes total sense!